src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -10,7 +12,14 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+
+    private UserService userService = new UserService();
+
     public boolean login(String email, String password) {
+        User userEmail = userService.findByEmail(email);
```

Comment:
Do you really need to create UserService() on each call of login or should we change declaration level for this variable?